cypress/integration/Dictionary/createDictionary.feature

```diff
@@ -0,0 +1,110 @@
+Feature: Dictionary Management
+
+  Scenario: A user to create a new dictionary
+    Given the user is logged in
+    And is on the user collections page
+    Then the user clicks on Create Dictionary Icon
+     Then the user should be able to create new dictionary
+
+  Scenario: A user should be able to create new dictionary by pressing on Create new dictionary icon
+    Given the user is on the Dictionaries page
+     When the user clicks on Create new dictionary icon
+     Then the create Dictionary form is filled
+     Then the user should be able to create new dictionary
+  
+  Scenario: A user should be able to create a new dictionary by filling the required information
+    Given the user is on the Create Dictionary page
+     When the user enters the required information in the form
+      And the user clicks the submit button
+     Then the user should be able to create a new dictionary
+  
+  Scenario: The Create Dictionary form should require a Dictionary Name to Create new dictionary
```

Comment:
It's best to use consistent words to describe scenarios. Since everything else is "A user can...", this should be something like "A user must supply a dictionary name".